CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 # OpenTelemetry Android Changelog
 
-## Version 0.7.0 (2024-08-14)
```

Comment:
Not sure why the old version was in there twice, but our new process uses a heading called `## Unreleased`